Classes/Controller/AbstractFluxController.php

```diff
@@ -245,6 +245,10 @@ protected function resolveOverriddenFluxControllerActionNameFromRequestParameter
      */
     protected function performSubRendering($extensionName, $controllerName, $actionName, $pluginSignature)
     {
+        if ($actionName == 'outlet') {
+            $input = array_replace(GeneralUtility::_GET($pluginSignature), GeneralUtility::_POST($pluginSignature));
```

Comment:
I'm not super happy with this part... our request-level arguments should already be in $this->request / $this->arguments of the controller. If not, that might indicate an issue with the scope of those arguments (e.g. wrong `pluginName` in form in Fluid).